Content.Client/GameTicking/Managers/ClientGameTicker.cs

```diff
@@ -19,18 +21,21 @@ namespace Content.Client.GameTicking.Managers
     public sealed class ClientGameTicker : SharedGameTicker
     {
         [Dependency] private readonly IStateManager _stateManager = default!;
-        [Dependency] private readonly IEntityManager _entityManager = default!;
+        [Dependency] private static readonly IEntityManager _entityManager = default!;
         [Dependency] private readonly IConfigurationManager _configManager = default!;
         [Dependency] private readonly SharedAudioSystem _audio = default!;
 
         [ViewVariables] private bool _initialized;
-        private Dictionary<NetEntity, Dictionary<string, uint?>>  _jobsAvailable = new();
+        private Dictionary<NetEntity, Dictionary<string, uint?>> _jobsAvailable = new();
         private Dictionary<NetEntity, string> _stationNames = new();
 
+        //cyberfinn Round-End Summary change
```

Comment:
You dont need to add your name everywhere git blame exists for this reason. Also while yes comments are good to include i feel like you are commenting a BIT too much on stuff that is self explaining,

Reply:
agreed lol, this is for me so that I can remember where I've made changes. This was my first time ever looking at the code for SS14, and I dont know anything about game dev, so needed to remind myself where I was making changes
Also, I'm used to commenting and overexplaining, it just helps those that dont know coding to check the logic and contribute by correcting or asking questions